libraries/AP_Beacon/AP_Beacon_Marvelmind.h

```diff
@@ -1,15 +1,33 @@
 /*
- * AP_Beacon_Marvelmind.h
- *
- *  Created on: 21.03.2017
+   This program is free software: you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation, either version 3 of the License, or
+   (at your option) any later version.
+
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+/*
+ Original C Code by Marvelmind (https://bitbucket.org/marvelmind_robotics/)
+ Adapted into Ardupilot by Karthik Desai, Amilcar Lucas
+ April 2017
  */
 
 #ifndef AP_BEACON_MARVELMIND_H_
 #define AP_BEACON_MARVELMIND_H_
 
 #pragma once
 
-#define AP_BEACON_MARVELMIND_MAX_STATIONARY_BEACONS 30
+// DF Logging currently only supports 4
+// EKF does support upto 10
+// Marvelmind supports upto 30
+// So supporting only 4 here
+#define AP_BEACON_MARVELMIND_MAX_STATIONARY_BEACONS 4
```

Comment:
Why beacon number is limited and hardcoded? Is it better to define the total number in a parameter ? kindly advise.